Configs/en-de_captions.yml

```diff
@@ -1,13 +1,13 @@
 # Model:
 model_name: ./Output/en-de_pipeline/bt_init/last
 load_entire_agent: false
-image_dim: 2048
```

Comment:
Are the changes to this config file (and other config files in this PR) intended to be specific to BEIT/live in a dedicated feature branch, or is this intended to be merged all into main?

If it is the latter, changing these values may impact the reproduced results of I2I. If we intend the pipeline to be run with or without BEIT (e.g. possible to run side-by-side), we may need to consider a new config to keep BEIT configs separate from the "generic" I2I configs?

Reply:
so basically we are not considering BEIT anymore. The `image_dim` here is changed the default to 768 because we want CLIP embeddings to be our default settings. But I am okay with just leaving 2048 as it is and probably also enabling overwritten functions so that we do not need to have separate config files for every small knob that we want to change.